lecture/8-cluster.tex

```diff
@@ -27,6 +27,10 @@
 	\end{frame}
 }
 
+% Special settings
+\setlength{\skip\footins}{0.2cm}
+\setlength{\footnotesep}{0.1cm}
```

Comment:
Comment "special settings" could be more elaborate (e.g. reduce spacing/margin of footlines). Additionally, seems relevant for all lecture slides, not only for clustering. Please move to the preamble.

Reply:
The comment in this case is not an actual comment, but rather a separator following the rest of the file (e.g. "%Outline" or "%Body"). In my opinion the lines are quite self-explanatory (it adjusts the spacing in the fullfootcites). But if it's not self-explanatory for you, I'm happy to add a comment in addition to the separator.

Regarding the preamble: Originally I had the settings there once. However, with this there is a risk that some already existing slide sets will be broken design-wise. That's why my suggestion would be that we transfer the settings to the preamble only in the preparation for the next semester, because then everyone can really test his slide sets with it and can check in detail whether there are unwanted shifts in the design.

Reply:
They are self-explanatory. The other "comments" are legacy which we simply migrated without removing them as we had other and more important tasks at hand such as restructuring and updating the content itself. I don't see any reason why we have to continue following this procedure anymore.

I highly doubt that moving the footnote settings to the preamble will break our design. However, that is something that's easy to check, isn't it? The worst case that can happen is that we have more space on slides.
Otherwise, we put it here and then have to create an issue that moves it later. So why not move it right now? We will go through each lecture chapter for the upcoming semester as well. Any problems regarding overflows or design breaks will be apparent then.

Reply:
A quick check on my side confirmed that we do not have any drastic design breaks when moving these settings to the preamble but indeed more space in each lecture slide that contains a footnote.

Reply:
Sorry, but I don't agree with you about the comments. I find that they bring some structure/make it easier to navigate through the document. So I definitely don't see these as legacy.

Regarding the settings: There may well be problems if, for example, graphics to refer to the relative height of the available space. Since more space is then available, this would then automatically also lead to a larger width. 

I can possibly spare the time to skim over the slide sets, but a detailed look is nothing I can do at the moment. In case of error correction - and thus another upload for this semester - I would ask you to take a close second look at slides with footnotes.

Reply:
I looked at all my slides and they seem fine. No need to re-uploading them.

Regarding image heights: Are we using relative heights somewhere? I am not, are you?